test_cli.py

```diff
@@ -1686,9 +1691,11 @@ def test(ctx):
     # Basic getting info and blob contents of a bundle
     path = test_path('a.txt')
     uuid = _run_command([cl, 'upload', path])
-    response = ctx.client.fetch_contents_info(uuid)
+    response = ctx.client.fetch_contents_info(BundleTarget(uuid, ''))
     check_equals(response['name'], uuid)
-    check_equals(open(path, 'rb').read(), ctx.client.fetch_contents_blob(uuid, '/').read())
+    check_equals(
+        open(path, 'rb').read(), ctx.client.fetch_contents_blob(BundleTarget(uuid, '/')).read()
```

Comment:
What's the difference between `subpath = ''` versus `subpath = '/'`?

Reply:
honestly I'm not sure. I don't think `subpath='/'` makes any sense, I didn't really attention to each individual target I encountered and generally wrapped them with the constructor. This can probably be an empty path.